File: src/graphql/searchById.ts
/**
 * GraphQL query to search for anime by ID
 */
export const SEARCH_ANIME_BY_ID = `
  query ($id: Int!) {
    Media(id: $id, type: ANIME) {
      id
      title { romaji english native }
      format
      status
      coverImage { large }
      siteUrl
    }
  }
`

File: go/internal/graphql/search_by_id.go
package graphql

// SearchAnimeByIDQuery is the GraphQL query for searching anime by ID
const SearchAnimeByIDQuery = `
	query ($id: Int!) {
		Media(id: $id, type: ANIME) {
			id
			title {
				romaji
				english
				native
			}
			format
			status
			coverImage {
				large
			}
			siteUrl
		}
	}`
